geom/type_null_geometry.go

```diff
@@ -24,7 +24,7 @@ func (ng *NullGeometry) Scan(value interface{}) error {
 // Value implements the database/sql/driver.Valuer interface.
 func (ng NullGeometry) Value() (driver.Value, error) {
 	if !ng.Valid {
-		return nil, nil
+		return nil, nil //nolint:nilnil
```

Comment:
`return nil, nil` is being detected as suspicious (usually this would be an error). It's not here, because the `database/sql/driver.Valuer` interface expects nil values (it interprets them as `NULL`).